openvdb/openvdb/tree/Tree.h

```diff
@@ -111,15 +115,31 @@ class OPENVDB_API TreeBase
     /// @brief Return the depth of this tree.
     ///
     /// A tree with only a root node and leaf nodes has depth 2, for example.
-    virtual Index treeDepth() const = 0;
+#if OPENVDB_ABI_VERSION_NUMBER >= 12
```

Comment:
I don't think we need to change `treeDepth()`, the standard tree depth is only 4 levels and it is never going to exceed the 32-bit limit.